src/popup/components/TransferReview.vue

```diff
@@ -1,10 +1,17 @@
 <template>
-  <div class="review-wrapper">
+  <div class="transfer-review">
     <ModalHeader
       :title="isMultisig ? $t('modals.multisigTxProposal.title') : $t('pages.send.reviewtx')"
       :subtitle="isMultisig ? null : $t('pages.send.checkalert')"
     />
 
+    <div
+      v-if="isMultisig"
```

Comment:
`v-show` instead of `v-if`

Reply:
hmm i always prefer v-if if i not need render something